persistence/sql/process_wait_until.go

```diff
@@ -0,0 +1,184 @@
+// Copyright 2023 XDBLab organization
```

Comment:
nit: if this file is for both ProcessWaitUntil and CompleteWaitUntil, probably rename the file to `wait_until.go` instead

Reply:
Done.